src/go/rpk/pkg/cli/cluster/health.go

```diff
@@ -60,6 +60,7 @@ following conditions are met:
 			}
 		},
 	}
+	p.InstallSASLFlags(cmd)
 	p.InstallAdminFlags(cmd)
```

Comment:
because in one admin is on top, in other sasl

Reply:
fixed